pom.xml

```diff
@@ -42,6 +42,8 @@
   <url>http://blueflood.io</url>
 
   <properties>
+    <VERSION_SUFFIX>0-SNAPSHOT</VERSION_SUFFIX>
+    <BLUEFLOOD_VERSION>2.0.${VERSION_SUFFIX}</BLUEFLOOD_VERSION>
     <maven.javadoc.skip>true</maven.javadoc.skip>
```

Comment:
They serve separate (bu related) purposes. The VERSION_SUFFIX is what gets overridden by the build script. The BLUEFLOOD_VERSION lets us modify the version for all submodules in a single place.